.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pyc
 .DS_Store
 _site
+.idea/
```

Comment:
These are IDE-specific files (pycharm). This line keeps my directory status clean when working with git.